frontend/public/index.html

```diff
@@ -5,10 +5,7 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
+    <meta name="description" content="Web site created using create-react-app" />
```

Comment:
Are you using prettier or another config to format your file,
 let's share one into this repo to make it a standard

Reply:
Yes I'm using the prettier configuration of VSCode, but we could have a `prettier` config, along with `husky` as a pre-commit hook so that we'll have the same configuration for each commit

Reply:
Yea let's do this part of #2